substrate/frame/contracts/fixtures/contracts/common/src/lib.rs

```diff
@@ -17,6 +17,8 @@
 #![no_std]
 #![cfg(any(target_arch = "wasm32", target_arch = "riscv32"))]
```

Comment:
We can remove this line now since we removed this crate from the workspace (as we should). It is never compiled for the host. Just disappearing the code instead of erroring is confusing. Maybe replace by an explicit compile error if it is compiled for anything else.